solidity/contracts/bancorx/interfaces/IBancorX.sol

```diff
@@ -1,6 +1,8 @@
 pragma solidity 0.4.26;
+import '../../token/interfaces/IERC20Token.sol';
 
 contract IBancorX {
+    function token() public view returns (IERC20Token) {this;}
```

Comment:
@yudilevi : We should try to avoid "dummy" implementation in interfaces.

Reply:
we'll clean it up when we move to explicit getters

Reply:
@yudilevi : Why not clean it up now?

Reply:
since it's pretty involved to do that across the board and it has no effect on public interface, so it's less urgent.